subworkflows/local/standardisation_profiles.nf

```diff
@@ -43,6 +47,23 @@ workflow STANDARDISATION_PROFILES {
         Standardise and aggregate
     */
 
+        // CENTRIFUGE
+
+    // Collect and replace id for db_name for prefix
+    // Have to sort by size to ensure first file actually has hits otherwise
+    // the script fails
+    ch_profiles_for_centrifuge = ch_input_profiles.centrifuge
+                                .map { [it[0]['db_name'], it[1]] }
+                                .groupTuple(sort: {-it.size()} )
```

Comment:
Is the `-it.size()` reverse sort?

Reply:
Yes, in this case makes it biggest to smallest (if I understood correctly). Makes it work though 😅